a3-live/src/App.js

```diff
@@ -27,25 +31,26 @@ function App() {
     this.state.loaded = true;
   */
   });
-  //TODO: refactor for use with hooks and mockData
-  //TODO: combine employees with teams and projects
+
 
   return (
     <React.Fragment>
       <NavigationBar />
 
-    <div className="container" id="main">
+    <Container maxWidth="sm">
       <div className="row">
-            {
-              this.state.Teams.data &&
-              this.state.Teams.data.map(
-                   team => {
-                    return <TeamInterface team={team} employeeList={Employees} project={project}/>
-                  }
+        {
+          mockTeams.map(
```

Comment:
If we're populating `Teams` state from `mockTeams`, then what's the reason to leave this hardcoded?
```suggestion
          Teams.map(
```